client/lib/paths/index.js

```diff
@@ -22,6 +22,6 @@ export function newPost( site ) {
 	return '/post' + sitePath;
 }
 
-export function createAccountUrl( { redirectTo } ) {
-	return `/start/account?redirect_to=${ redirectTo }`;
+export function createAccountUrl( { redirectTo, ref } ) {
```

Comment:
I've confirmed that this PR updates all `createAccountUrl` usages to add a `ref` argument 👍